FusionIIIT/applications/placement_cell/views.py

```diff
@@ -2608,17 +2609,30 @@ def cv(request, username):
     # print(achievementcheck,' ',educationcheck,' ',publicationcheck,' ',patentcheck,' ',internshipcheck,' ',projectcheck,' \n\n\n')
     user = get_object_or_404(User, Q(username=username))
     profile = get_object_or_404(ExtraInfo, Q(user=user))
+    student_info=get_object_or_404(Student,Q(id=user.username))
+    # print("Student------>",vars(student_info))
+    batch=student_info.batch
+    # print("Year----->",user.date_joined.year)
+    # print("User----->",vars(user))
+    # print("Profile---->",vars(profile))
     now = datetime.datetime.now()
-    if int(str(profile.id)[:2]) == 20:
-        if (now.month>4):
-          roll = 1+now.year-int(str(profile.id)[:4])
-        else:
-          roll = now.year-int(str(profile.id)[:4])
+    print("year----->",now.year)
+    if now.year-batch<=4:
+        roll=now.year-batch
     else:
-        if (now.month>4):
-          roll = 1+(now.year)-int("20"+str(profile.id)[0:2])
-        else:
-          roll = (now.year)-int("20"+str(profile.id)[0:2])
+        roll=4
+        
+    # if int(str(profile.id)[:2]) == 20:
+    #     if (now.month>4):
+    #       roll = 1+now.year-int(str(profile.id)[:4])
+    #     else:
+    #       roll = now.year-int(str(profile.id)[:4])
+    # else:
+    #     if (now.month>4):
+    #       roll = 1+(now.year)-int("20"+str(profile.id)[0:2])
+    #     else:
+    #       roll = (now.year)-int("20"+str(profile.id)[0:2])

```

Comment:
```suggestion
    # if int(str(profile.id)[:2]) == 20:
    #     if (now.month>4):
    #       roll = 1+now.year-int(str(profile.id)[:4])
    #     else:
    #       roll = now.year-int(str(profile.id)[:4])
    # else:
    #     if (now.month>4):
    #       roll = 1+(now.year)-int("20"+str(profile.id)[0:2])
    #     else:
    #       roll = (now.year)-int("20"+str(profile.id)[0:2])

```

Kindly remove these comments if not needed.